### ISSUE TITLE
[A11y] Advanced Search - Link purpose

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1798]
### Operating System

Windows

### Browser

Edge

### Affected page

Advanced Search

### Admin user?

- [X] Yes

### Issue Description

- There's a link that's just "@oregondigcol" - there's no context to make it clear this is a link to the Oregon Digital twitter feed
- There's an empty `<a>` tag used as the target for the skip link.  This should probably just be a div or something.  I don't know if it's a WCAG failure, it just confuses Accessibility Insights and might be a constant nuisance when testing things like this.

### Suggested Fix

- Probably best to make "@oregondigcol" into something like "OD on Twitter" to help all users.  Most people will recognize the twitter symbol, but cognitively impaired people may not, people who haven't used twitter much may not, and blind people obviously need something more than a decorative image.
- Make the empty `<a>` tag a div or maybe even just make the `main` region be the target of the skip link

**Comment by jechols** [comment]
Closing and splitting into two tickets
